apps/web/src/pages/TalentNominations/NominationGroupEvaluationDialog/components/LateralMovementSection.tsx

```diff
@@ -0,0 +1,251 @@
+    }
+
+    if (
+      selectedLateralMovementDecision !== TalentNominationGroupDecision.Rejected
+    ) {
+      resetDirtyField("lateralMovementRejectedNotes", null);
+    }
+  }, [resetField, selectedLateralMovementDecision]);
+
+  const talentNominationGroup = getFragment(
+    NominationGroupEvaluationDialogLateralMovement_Fragment,
+    talentNominationGroupQuery,
+  );
+
+  const talentNominationGroupOptions = getFragment(
+    NominationGroupEvaluationDialogLateralMovementOptions_Fragment,
+    talentNominationGroupOptionsQuery,
+  );
+  const nominations = talentNominationGroup.nominations ?? [];
+  const lateralMovementOptionValuesInThisNominationGroup = nominations
+    .filter((n) => n.nominateForLateralMovement)
+    .flatMap((n) => n.lateralMovementOptions)
+    .map((o) => o?.value)
+    .filter(notEmpty);
+  // I don't actually need a set of options but the sort is handy
+  const lateralMovementListItems = localizedEnumToOptions(
+    talentNominationGroupOptions?.talentNominationLateralMovementOptions,
+    intl,
+    [
+      TalentNominationLateralMovementOption.SmallDepartment,
+      TalentNominationLateralMovementOption.LargeDepartment,
+      TalentNominationLateralMovementOption.CentralDepartment,
+      TalentNominationLateralMovementOption.NewDepartment,
+      TalentNominationLateralMovementOption.ProgramExperience,
+      TalentNominationLateralMovementOption.PolicyExperience,
+      TalentNominationLateralMovementOption.Other,
+    ],
+  ).map((option) => ({
+    key: option.value,
+    value: lateralMovementOptionValuesInThisNominationGroup.includes(
+      option.value as TalentNominationLateralMovementOption,
+    ),
+    label: option.label,
+  }));
+
+  const lateralMovementOptionsOthersWithDuplicates = nominations
+    .filter((n) => n.nominateForLateralMovement)
+    .map((n) => n.lateralMovementOptionsOther)
+    .filter(notEmpty);
+  const lateralMovementOptionsOthers = uniq(
+    lateralMovementOptionsOthersWithDuplicates,
+  );
+  lateralMovementOptionsOthers.sort((a, b) => a.localeCompare(b));
+
+  return (
+    <div
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(column)"
+      data-h2-gap="base(x1)"
+    >
+      <Heading
+        level="h3"
+        size="h6"
+        data-h2-margin="base(0)"
+        data-h2-font-weight="base(normal)"
+      >
+        {intl.formatMessage({
+          defaultMessage: "Nomination for lateral movement",
+          id: "rPUWP5",
+          description: "heading for lateral movement nomination section",
+        })}
+      </Heading>
+      <FieldDisplay
+        label={intl.formatMessage(
+          talentNominationGroupMessages.lateralMovementOptions,
+        )}
+      >
+        {lateralMovementListItems.map((item) => (
+          <li
+            key={item.key}
+            data-h2-display="base(flex)"
+            data-h2-align-items="base(flex-start)"
+            data-h2-gap="base(x.25)"
```

Comment:
I could be wrong here but I think these were added for the icon? If so, that should be handled by the `BoolChecIcon` component

Reply:
Oh, you're right!
[remove unnecessary boolcheckicon styling](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13305/commits/1f84f1c20c8cbab93418276371bcee218bad22c3)